lib/plottr_import_export/src/exporter/word/exporters/places.js

```diff
@@ -18,48 +17,50 @@ export default function exportPlaces(state, doc, options) {
     )
   }
 
-  const paragraphs = places(
-    state.places,
-    state.customAttributes['places'],
-    state.images,
-    doc,
-    options
-  )
+  const paragraphs = places(state.places, state.customAttributes['places'], state.images, options)
 
-  return { children: children.concat(paragraphs) }
+  return [{ children: children.concat(paragraphs) }]
 }
 
-function places(places, customAttributes, images, doc, options) {
+function places(places, customAttributes, images, options) {
   let paragraphs = []
   places.forEach((pl) => {
-    paragraphs.push(new Paragraph(''))
+    paragraphs.push(new Paragraph({ text: '' }))
     paragraphs.push(new Paragraph({ text: pl.name, heading: HeadingLevel.HEADING_2 }))
     if (options.places.images && pl.imageId) {
       const imgData = images[pl.imageId] && images[pl.imageId].data
       if (imgData) {
-        const image = Media.addImage(
-          doc,
-          Buffer.from(imgData.replace('data:image/jpeg;base64,', ''), 'base64')
+        paragraphs.push(
+          new Paragraph({
+            children: [
+              new ImageRun({
+                data: imgData,
+                transformation: {
+                  width: 300,
+                  height: 300,
+                },
+              }),
+            ],
+          })
         )
-        paragraphs.push(new Paragraph({ children: [image] }))
       }
     }
     if (options.places.descriptionHeading) {
       paragraphs.push(new Paragraph({ text: t('Description'), heading: HeadingLevel.HEADING_3 }))
     }
     if (options.places.description) {
-      paragraphs.push(new Paragraph(pl.description))
+      paragraphs.push(new Paragraph({ text: pl }.description))
```

Comment:
@Quiescent im not sure if this is intentional but i haven't encountered a format like this `{ text: pl }.description)` :sweat_smile: